appveyor.yml

```diff
@@ -34,7 +34,7 @@ build_script:
 
 
 on_success:
-  - Rscript -e "covr::codecov()"
+  - Rscript -e "covr::codecov(function_exclusions = c('print\\.', '\\.onLoad'))"

```

Comment:
Remove those function from coverage. Does not make sense as we cannot test them (and onLoad is called no matter what)